### ISSUE TITLE
Trouble with usage

### ISSUE THREAD

**Issue by alehandrof** [lesaff/statamic-date_calc#2]
If I enter this:

```
{{ session__date }} {{ session__time__start }} |
{{ date_calc start_date="{ session__date } { session__time__start }" method="addMinutes" value="0" }} |
{{ date_calc start_date="2015-05-27 10:30" method="addMinutes" value="0" }}
```

The output is this:

> 2015-05-27 10:30 | | 2015-05-27 10:30:00

Passing the two variables (1st line) to date_calc (2nd line) should produce the same output as the hard-coded tag (3rd line). But it doesn't. The 2nd line doesn't produce any output whatsoever (it would have been the two pipes).

**Comment by lesaff** [comment]
Let me give this a try on my test site.

**Comment by lesaff** [comment]
Just tested the following on my test site and it works. I am using the Statamic built-in `{{ current_date }}` for my dynamic variable testing source.

```
From variable: {{ date_calc start_date="{ current_date }" method="addMinutes" value="10" }}<br />
Hard coded: {{ date_calc start_date="2015-05-27 10:30" method="addMinutes" value="10" }}
```

Question: How do you create the `{{ session__date }}` variable? What is the format?

**Comment by alehandrof** [comment]
Something interesting is going on. I just copied & pasted your testing values and the "from variable" is blank on my site.

If it's still useful, `{{ session__date }}` and `{{ session__time__start }}` correspond to `2015-05-27` and `10:30`, respectively. The first is a date fieldtype; the latter is a text fieldtype (so that I can enter the time in 24-hour format; Statamic seems not to support that otherwise).

**Comment by alehandrof** [comment]
`{{ current_date }}` by itself outputs what you'd expect it to ("2015-03-08").

**Comment by lesaff** [comment]
Ok, that's good to know. I will try to simulate your case on my test site.

**Comment by alehandrof** [comment]
Thanks. No rush, BTW. Let me know if there's any logging I can provide that might be useful.

**Comment by lesaff** [comment]
I'll get back to you sometime today. Just need to run some errands.

**Comment by lesaff** [comment]
OK, just tested using system variable, custom variables and hard coded value.

```
{{# Declare custom variables #}}
{{ var:session_date is="2014-03-06" }}
{{ var:session_time is="23:55" }}

Custom variable output: {{ var:session_date }} {{ var:session_time }}<br />

From system variable: {{ date_calc start_date="{ current_date }" method="addMinutes" value="10" }}<br />

From custom variable: {{ date_calc start_date="{ var:session_date } { var:session_time }" method="addMinutes" value="10" }}<br />

Hard coded: {{ date_calc start_date="2015-05-27 10:30" method="addMinutes" value="10" }}
```

Output looks like this

```
Custom variable output: 2014-03-06 23:55
From system variable: 2015-03-08 00:10:00
From custom variable: 2014-03-07 00:05:00
Hard coded: 2015-05-27 10:40:00
```

I am using Mike Hue's [statamic_var](https://github.com/michaelhue/statamic-var) addon to help me create custom variables on the fly.

I use 23:55 as the custom time input to check whether it takes 24 hr format or not and yes, it did.

**Comment by lesaff** [comment]
Try using my example above and see if you're getting any output. Also, I am testing it on Statamic 1.10.3.

**Comment by alehandrof** [comment]
I tried it on vanilla Statamic and it looks fine. I should have thought of that before bothering you. Sorry about that.

I think my site's having some kind of broader issue. I just noticed that variables are not passing into partials. I'm wondering if the two issues are related. But I'm too beat to figure it out now.

Thank you for helping out. Clearly the trouble is with my site. I'll close the issue.
